corehq/motech/generic_inbound/views.py

```diff
@@ -180,6 +181,8 @@ def generic_inbound_api(request, domain, api_id):
         return JsonResponse({'error': str(e)}, status=500)
     except UserError as e:
         return JsonResponse({'error': str(e)}, status=400)
+    except GenericInboundRequestFiltered:
+        return JsonResponse({}, status=204)
```

Comment:
Once this is reconciled with the logging PR, we'll have to remember to capture this status appropriately in the log